recipes/orcania/all/conanfile.py

```diff
@@ -0,0 +1,141 @@
+from conan import ConanFile
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rmdir, save
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.microsoft import is_msvc
+import os
+import textwrap
+
+required_conan_version = ">=1.52.0"
+
+
+class OrcaniaConan(ConanFile):
+    name = "orcania"
+    description = "Potluck with different functions for different purposes that can be shared among C programs"
+    homepage = "https://github.com/babelouest/orcania"
+    topics = ("utility", "functions", )
+    license = "LGPL-2.1-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "base64url": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "base64url": True,
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
```

Comment:
```suggestion
            try:
                del self.options.fPIC
            except Exception:
                pass
```

Reply:
Addressed in https://github.com/conan-io/conan-center-index/pull/13589/commits/ca1863cb824f2b86cae211f60e9fc8da96aa7c0d